Documentation/platforms/aws.md

```diff
@@ -0,0 +1,77 @@
+## Introduction
+
+This guide assumes operational knowledge of Amazon Web Services (AWS), specifically Amazon Elastic Compute Cloud (EC2).
```

Comment:
add something a bit more here, like the goal of this doc / what it's meant to communicate?